src/System.Device.Gpio.Tests/GpioControllerTestBase.cs

```diff
@@ -95,7 +95,22 @@ public void IsPinOpenTest()
                 controller.ClosePin(LedPin);
                 Assert.False(controller.IsPinOpen(LedPin));
             }
-        }
+			using (GpioController controller = new GpioController(GetTestNumberingScheme(), GetTestDriver()))
```

Comment:
can you create a separate test case? If using `using` statement twice is needed to repro please still do it separately

Reply:
Created as a separate test case.